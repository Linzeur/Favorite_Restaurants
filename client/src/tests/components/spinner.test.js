import React from "react";
import { render } from "@testing-library/react";

import Spinner from "../../components/spinner";

test("spinner component", () => {
  const { asFragment } = render(<Spinner />);

  expect(asFragment()).toMatchSnapshot();
});
