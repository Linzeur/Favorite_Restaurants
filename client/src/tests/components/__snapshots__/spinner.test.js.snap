// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`spinner component 1`] = `
<DocumentFragment>
  <div
    class="css-w40flo-Spinner"
  >
    <div
      class="css-lehx5n-Spinner"
    />
    <div
      class="css-wk0iwg-Spinner"
    />
    <div
      class="css-1xsj7ka-Spinner"
    />
  </div>
</DocumentFragment>
`;
